antarest/study/business/areas/thermal_management.py

```diff
@@ -414,7 +414,7 @@ def duplicate_cluster(
         study: Study,
         area_id: str,
         source_id: str,
-        new_cluster_name: str,
+        new_cluster_name: LowerCaseStr,
```

Comment:
Here too I think it should be removed be cause it does not actually performs the transformation